gravitee-apim-e2e/lib/fixtures/management/ApisFaker.ts

```diff
@@ -131,4 +132,26 @@ export class ApisFaker {
       ...attributes,
     };
   }
+
+  static updateFromApiEntity(apiEntity: ApiEntity, attributes?: Partial<UpdateApiEntity>): UpdateApiEntity {
```

Comment:
Naming suggestion;

As the actual fakers functions are :
- `api` returning ApiEntity
- `newApi` returning NewApiEntity
- ..etc..

Should this one be named `updateApi`returning UpdateApiEntity ?